whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapConformance.java

```diff
@@ -0,0 +1,15 @@
+package net.ripe.db.whois.api.rdap;
+
+public enum RdapConformance {
```

Comment:
I think this enum is not being used now? There is already a constant list for this?

Reply:
we are using this enum everywhere instead of in the tests (Ed told me to use directly the profile to make the tests more clear)